jetbrains-ultimate/src/software/aws/toolkits/jetbrains/services/lambda/nodejs/NodeJsLambdaHandlerResolver.kt

```diff
@@ -92,6 +99,15 @@ class NodeJsLambdaHandlerResolver : LambdaHandlerResolver {
         return true
     }
 
+    /**
+     * Whether the element is top level PSI element for a valid Lambda handler. It must be in the format as:
+     * export const lambdaHandler = functionExpression
+     */
+    private fun PsiElement.isValidTypeScriptLambdaHandler(): Boolean =
```

Comment:
does this work with
```ts
export lambdaHandler() {
...
}
```
or is that actually invalid?

Reply:
it's invalid TS, so that shouldn't be an issue

Reply:
TS compiles this down to
```
async fucntion handler(event) {
...
};
exports.handler = handler;
```
and SAM CLI seems happy. will add